Domains/4-Application/OpenSite/OpenSite.ecschema.xml

```diff
@@ -433,6 +443,16 @@
         </Target>
     </ECRelationshipClass>
 
+    <ECRelationshipClass typeName="DrivewayMidVertexOwnsTransition" strength="embedding" strengthDirection="Forward" modifier="Sealed">
+        <BaseClass>bis:ElementOwnsChildElements</BaseClass>
+        <Source multiplicity="(1..1)" roleLabel="owns" polymorphic="false">
+            <Class class="DrivewayMidVertex" />
+        </Source>
+        <Target multiplicity="(2..2)" roleLabel="is owned by" polymorphic="false">
+            <Class class="DrivewayTransition" />
+        </Target>
+    </ECRelationshipClass>
+
```

Comment:
I don't think we want to make a change and immediately release a new Schema version - we need to test this schema first. I think normally, you make changes to the WIP (work in progress) schema, new `dev` version of this schema is published to NPM, we test it, perhaps make adjustments, and then we make a release PR.